routes/matches/matches.js

```diff
@@ -719,15 +756,18 @@ router.get("/limit/:limiter", async (req, res, next) => {
     let sql;
     if (req.user !== undefined && Utils.superAdminCheck(req.user)) {
       sql =
-        "SELECT * FROM `match` WHERE cancelled = 0 OR cancelled IS NULL ORDER BY end_time DESC LIMIT ?";
+        "SELECT mtch.* FROM `match` mtch WHERE mtch.cancelled = 0 OR mtch.cancelled IS NULL ORDER BY mtch.end_time DESC LIMIT ?";
```

Comment:
How come this one doesn't have game server info, but the non-user statement does? Also see above for SQL statement.